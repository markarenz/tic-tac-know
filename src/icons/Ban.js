import React from 'react';

const Ban = () => {
  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 122.88 122.88"
      style={{ margin: '0 auto' }}
    >
      <path
        d="M61.44 0c33.926 0 61.44 27.514 61.44 61.44s-27.514 61.439-61.44 61.439C27.513 122.88 0 95.366 0 61.44S27.513 0 61.44 0zm37.065 37.049L37.051 98.503a44.14 44.14 0 0024.389 7.307c24.5 0 44.369-19.869 44.369-44.369 0-9.01-2.69-17.391-7.304-24.392zm-74.127 48.78l61.453-61.454a44.155 44.155 0 00-24.391-7.304c-24.5 0-44.369 19.87-44.369 44.369a44.15 44.15 0 007.307 24.389z"
        fill="#ee4444"
      ></path>
    </svg>
  );
};

export default Ban;
