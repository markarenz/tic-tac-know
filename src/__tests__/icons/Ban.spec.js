import { render, screen } from '@testing-library/react';
import Ban from '../../icons/Ban';

describe('Ban', () => {
  it('renders component', () => {
    const container = render(<Ban />);
    expect(container).toMatchSnapshot();
  });
});
