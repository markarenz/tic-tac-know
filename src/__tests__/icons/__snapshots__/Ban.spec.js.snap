// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ban renders component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        style="margin: 0px auto;"
        viewBox="0 0 122.88 122.88"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M61.44 0c33.926 0 61.44 27.514 61.44 61.44s-27.514 61.439-61.44 61.439C27.513 122.88 0 95.366 0 61.44S27.513 0 61.44 0zm37.065 37.049L37.051 98.503a44.14 44.14 0 0024.389 7.307c24.5 0 44.369-19.869 44.369-44.369 0-9.01-2.69-17.391-7.304-24.392zm-74.127 48.78l61.453-61.454a44.155 44.155 0 00-24.391-7.304c-24.5 0-44.369 19.87-44.369 44.369a44.15 44.15 0 007.307 24.389z"
          fill="#ee4444"
        />
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      style="margin: 0px auto;"
      viewBox="0 0 122.88 122.88"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M61.44 0c33.926 0 61.44 27.514 61.44 61.44s-27.514 61.439-61.44 61.439C27.513 122.88 0 95.366 0 61.44S27.513 0 61.44 0zm37.065 37.049L37.051 98.503a44.14 44.14 0 0024.389 7.307c24.5 0 44.369-19.869 44.369-44.369 0-9.01-2.69-17.391-7.304-24.392zm-74.127 48.78l61.453-61.454a44.155 44.155 0 00-24.391-7.304c-24.5 0-44.369 19.87-44.369 44.369a44.15 44.15 0 007.307 24.389z"
        fill="#ee4444"
      />
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
